src/core/product-transformer.ts

```diff
@@ -71,6 +71,35 @@ namespace Utils {
       return Utils.remap(transformed, effectiveStructure, baseProduct);
     };
   }
+
+  export const transformData = (data: any) => {
+    return ({
+      ...data,
+      price: transformPrice(data.price),
+    });
+  };
+
+  // note that this assumes data won't have thousands separators
```

Comment:
This will work even with thousands separators because `*` is greedy.

Reply:
Yeah I should've clarified, 100,000 is when there are issues :(